Descent3/OsirisLoadandBind.cpp

```diff
@@ -2683,7 +2683,7 @@ bool Osiris_RestoreSystemState(CFILE *file) {
     return false;
   }
 
-  uint8_t version = (uint8_t)cf_ReadByte(file);
+  // uint8_t version = (uint8_t)cf_ReadByte(file);
```

Comment:
Is it safe to remove the byte read? All subsequent reads will at an other position than before.

Reply:
Good catch, I fixed it now.